tests/v2/unit/adapters/torch/lightning/test_helpers.py

```diff
@@ -71,33 +75,51 @@ def generate_visual_prompting_dataset(use_mask: bool = False) -> DatasetEntity:
             use_mask_as_annotation=use_mask,
         )
 
-        out_shapes = []
+        annotations = []
         for shape in shapes:
             shape_labels = shape.get_labels(include_empty=True)
 
             in_shape = shape.shape
             if use_mask:
                 if isinstance(in_shape, Image):
-                    out_shapes.append(shape)
+                    ann = DatumMask(
+                        image=in_shape.numpy,
+                        label=shape_labels[0].id,
+                    )
+                    annotations.append(ann)
             else:
                 if isinstance(in_shape, Rectangle):
                     points = [
-                        Point(in_shape.x1, in_shape.y1),
-                        Point(in_shape.x2, in_shape.y1),
-                        Point(in_shape.x2, in_shape.y2),
-                        Point(in_shape.x1, in_shape.y2),
+                        in_shape.x1 * 640,
+                        in_shape.y1 * 480,
+                        in_shape.x2 * 640,
+                        in_shape.y1 * 480,
+                        in_shape.x2 * 640,
+                        in_shape.y2 * 480,
+                        in_shape.x1 * 640,
+                        in_shape.y2 * 480,
                     ]
                 elif isinstance(in_shape, Ellipse):
-                    points = [Point(x, y) for x, y in in_shape.get_evenly_distributed_ellipse_coordinates()]
+                    points = []
```

Comment:
are these values int or float? Would be good to know when reading?

Reply:
I clarified this to `points: list[float] = []` at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.